lib/constants.go

```diff
@@ -458,7 +458,7 @@ func (params *BitCloutParams) EnableRegtest() {
 
 	// Add a key defined in n0_test to the ParamUpdater set when running in regtest mode.
 	// Seed: verb find card ship another until version devote guilt strong lemon six
-	params.ParamUpdaterPublicKeys[MakePkMapKey(MustBase58CheckDecode("tBCKVERmG9nZpHTk2AVPqknWc1Mw9HHAnqrTpW1RnXpXMQ4PsQgnmV"))] = true
+	params.ParamUpdaterPublicKeys[MakePkMapKey(MustBase58CheckDecode("tBCKW665XZnvVZcCfcEmyeecSZGKAdaxwV2SH9UFab6PpSRikg4EJ2"))] = true
```

Comment:
let's revert the public key. I didn't realize there was the seed above it for testnet.